src/feature.rs

```diff
@@ -147,7 +147,7 @@ fn expand_scenario(
     //       https://github.com/Canop/lazy-regex/issues/10
     #[expect(clippy::unwrap_used, reason = "regex is valid")]
     static TEMPLATE_REGEX: LazyLock<Regex> =
-        LazyLock::new(|| Regex::new(r"<([^>\s]+)>").unwrap());
+        LazyLock::new(|| Regex::new("<([^>]+)>").unwrap());
```

Comment:
@aepfli allowing spaces seems ok, but does it mean we should also allow `\t`, `\r`, `\n` and other "brothers"?

I think it's only meaningful to allow spaces and tabs.